plugin/build_baron.go

```diff
@@ -82,7 +82,7 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 					template.HTML(`<script type="text/javascript" src="/static/plugins/buildbaron/js/task_build_baron.js"></script>`),
 				},
 				DataFunc: func(context UIContext) (interface{}, error) {
-					_, enabled := bbp.opts.Projects[context.ProjectRef.Identifier]
+					enabled := (len(bbp.opts.Projects[context.ProjectRef.Identifier].TicketSearchProjects) > 0)
```

Comment:
I think the outermost parenthesis are extra

Reply:
removed it, thanks!